### PR TITLE
feat: add useActionSheet hook

### PR DESCRIPTION
**Pull request by SimenB**
If you don't want the hook, could we export the context so I can add a hook in my own code?

### CODE REVIEW

**Comment by SimenB** [comment]
@brentvatne thoughts on this? 🙏

**Comment by tuanna1601** [comment]
yes, we definitely need this, please merge it 🙇‍♂️

File: README.md [hunk 1]
```diff
@@ -55,6 +55,17 @@ _onOpenActionSheet = () => {
 };
 ```
 
+You can use a hook instead of the higher order component if you are on React 16.6 or newer.
```
Comment (SimenB) [inline README.md]:
```suggestion
You can use a hook instead of the higher order component if you are on React 16.8 or newer.
```

**Comment by Naturalclar** [comment]
Definitely would love to have this feature :heart:, any plan on having this merged?

**Comment by douglaszaltron** [comment]
yup!

**Comment by brentvatne** [comment]
sorry for the delay, this looks good to me!

would anyone like to help out with maintaining this project? send me an email at brent@expo.io if so

**Pull request merged**